components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -5325,27 +5325,20 @@ public void updatePermissionListOfGroup(String groupName, String[] permissionToA
         return filteredFlatAttributeMap;
     }
 
-    private Map<String, Attribute> getFilteredEnterpriseUserSchemaAttributes(Map<ExternalClaim, LocalClaim>
+    /**
+     * Get filtered claims based on property `isSupportedByDefault` of mapped local dialect claim.
+     *
+     * @param scimClaimToLocalClaimMap
+     * @return
```

Comment:
```suggestion
     * @return List of filtered attributes
```

Reply:
Fixed in commit e0f62727dd4fc285db2aa18b57836d5b623544d5

Reply:
You fixed for `getFilteredUserSchemaAttributes` method :)

Reply:
silly mistake , resolved in commit 176fdfcc6a4d814372524205a649b8dff8556876